src/DynamoCore/Library/FunctionDescriptor.cs

```diff
@@ -172,7 +172,7 @@ public FunctionDescriptor(FunctionDescriptorParams funcDescParams)
 
         public string Summary
         {
-            get { return summary ?? (summary = this.GetSummary(pathManager)); }
+            get { return summary ?? (summary = this.GetSummary()); }
```

Comment:
If `TypedParameter` no longer requires `pathManager` (see below), then do we need this in `FunctionDescriptor` class? I know we have a public property `FunctionDescriptor.PathManager`, not sure if that's used anywhere. But please have a look, thanks! :)